Assets/Scripts/InventoryManager.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.InputSystem;
+
+public class InventoryManager : MonoBehaviour
+{
+    public static InventoryManager Instance;
+
+    //This will be the list that things are added and removed to via purchasing items
+    [SerializeField] private List<PlaceableObject> inventoryPlaceableObjects = new List<PlaceableObject>();
+    [SerializeField] private GameObject inventoryGUI;
+    [SerializeField] private GameObject inventoryGrid;
+    [SerializeField] private GameObject inventoryItemPrefab;
+    [SerializeField] private GameObject playerHeldItemParent;
+    [SerializeField] private float stockScale;
+    [SerializeField] private float structureScale;
+
+
+    private PlaceableObject heldObject;
+    public PlaceableObject HeldObject { get => heldObject; set => heldObject = value; }
+    private int tabIndex;
+    private List<GameObject> gridObjectDisplayList = new List<GameObject>();
+    //List of items currently displayed in GUI
+    private List<PlaceableObject> inventoryObjectDisplayList = new List<PlaceableObject>();
```

Comment:
any instance that needs `new [Type]()` can be condensed to `new()`
but that might be confusing to others